bin/martinize2

```diff
@@ -188,6 +188,11 @@ def write_gmx_topology(system, top_path, defines=(), header=()):
             # molecules, without being uninterupted by other moltypes. Even in
             # that case, we want to write the ITP only once.
             with open('{}.itp'.format(moltype), 'w') as outfile:
+                # here we format and merge all citations
+                for citation in molecule.citations:
+                    cite_dict = molecule.force_field.citations[citation]
+                    cite_string = " ".join([field for field in cite_dict.values()])
+                    header.append(cite_string + "\n")
```

Comment:
Unless we do actual bibtex interpretation and consequent formatting the citations will be hard to read/interpret/ugly. This is one reason why I would prefer to put formatted strings as the citations rather than bibtex in the FF files

Reply:
this is just to check that the citation actually arrives in the itp file. Formatting is easy based on bibtex which is the entire point of doing it in the first place.